aries-site/src/pages/templates/internationalization.mdx

```diff
@@ -0,0 +1,97 @@
+
+## Overview
+
+Internationalizion (I18N) and Localization (L10N) are important for web content
+and applications which are intended to be used by global users. It's even important
+for users in a single location who simply speak different languages.
+
+Internationalization is the process of making your application tailor its
+content to different languages or even locations. Localization is the process of
+translating content and behavior for a particular language or location.
+
+Even if you don't localize the messages and content of your application or site in a
+different language, users can benefit from your site being internationalized. Part
+of internationalizating is making your site recognize the locale set in the browser
+and format things like dates and times, currency and numbers specifically
+for that locale. Browsers have great built-in support for formatting that content
+appropriately for the locale. For example, a user in Australia or the United Kingdom
+may be perfectly happy with your content being in English but would much prefer any
+dates or times be presented with the day of the month before the month
+(28/06/2021 versus 6/28/2021). This is easy to do with `Date.toLocalDateString()`.
+More information on browser and javsascript localization support is available from
+many sources such as ***TODO***
+
+## Guidelines
+
+Depending on how your site or application is implemented, you'll need to pick
+an appropriate mechanism for choosing a language and locale and then provide
+content for the language and locale the user chose. You can provide a way to pick
+a language or simply try to use the preference set by the browser. Browsers include
```

Comment:
Maybe rewording this section to something like, "When possible, set localize content based on the locale preferences set by the user or returned in the header of the page request."